src/elements/JobsPanelTitle/JobsPanelTitleView.js

```diff
@@ -38,6 +38,17 @@ const JobsPanelTitleView = ({
     'job-panel__title-edit-icon',
     openScheduleJob && 'job-panel__title-edit-icon_disabled'
   )
+  const titleValidationTip = editTitle ? (
+    <>
+      <span>&bull; Valid characters: A-Z, a-z, 0-9, -, _, .</span>
+      <br />
+      <span>&bull; Must being and end with: A-Z, a-z, 0-9</span>
```

Comment:
Oops, it's my typo in the task description.
Please change to "begin".
```suggestion
      <span>&bull; Must begin and end with: A-Z, a-z, 0-9</span>
```